sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj

```diff
@@ -18,7 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault.Core" />
-    <PackageReference Include="WindowsAzure.Storage" />
```

Comment:
I wonder if we can remove this from the central package versions as well.   I'll follow-up on this.

Reply:
Thx Jesse